glance/glance.go

```diff
@@ -106,3 +106,57 @@ func (c *Client) GetImageDetail(imageId string) (*ImageDetail, error) {
 	}
 	return &resp.Image, nil
 }
+
+// ImageDetailV2 describes extended information about an image for API v2.0.
```

Comment:
The url is at the top of the page.  I'm adding one for the image functions utilizing the compute server as well.